README.md

```diff
@@ -266,109 +245,52 @@ class MyElement extends
-
-Also note that the Metadata Reflection polyfill is **47 KB** (7 KB gzipped), so
-be sure to consider strongly whether the cost of shipping this polyfill is
-worth the convenience for your project.
-
-Without Metadata Reflection, the Polymer property type must be passed
-explicitly to the decorator factory, because type information is not otherwise
-available at runtime:
-
-```ts
-@property({type: String})
-myProperty: string;
-```
-
-With Metadata Reflection, the TypeScript type annotation alone is sufficient,
-because the compiler will emit type information that the decorator can use to
-automatically set the Polymer property type:
-
-```ts
-@property()
-myProperty: string;
-```
-
-To enable Metadata Reflection:
-
-1. Enable the
-   [`emitDecoratorMetadata`](https://www.typescriptlang.org/docs/handbook/decorators.html#metadata)
-   TypeScript compiler setting. Use the `--emitDecoratorMetadata` flag, or
-   update your `tsconfig.json` to include:
-
-   ```js
-   {
-     "compilerOptions": {
-       "emitDecoratorMetadata": true
-     }
-   }
-   ```
-
-2. Install the Metadata Reflection API runtime polyfill from
-   [rbuckton/reflect-metadata](https://github.com/rbuckton/reflect-metadata):
-
-   ```sh
-   bower install --save rbuckton/reflect-metadata
-   ```
-
-3. Load the polyfill at the top-level of your application, and in your tests:
-
-   ```html
-   <script src="/bower_components/reflect-metadata/Reflect.js"></script>
-   ```
-
-
 ## FAQ
 
 ### Do I need this library to use Polymer and TypeScript?
-No, you can also use Polymer and TypeScript without additional client
-libraries. As of Polymer 2.4, TypeScript type declarations are available in the
-[`types/`](https://github.com/Polymer/polymer/tree/master/types) directory. The
-advantage of using these decorators are additional type safety and convenience.
-For simple elements and applications, it may be preferable to use the vanilla
-Polymer API, like this:
+No, you can also use Polymer and TypeScript without any additional libraries.
+Polymer 3.0 ships with declarations to let you use the API with TypeScript
+directly.  The advantage of using these decorators are additional type safety
+and convenience. For simple elements and applications, it may be preferable to
+use the vanilla Polymer API, like this:
 
 ```ts
-/// <reference path="./bower_components/polymer/types/polymer-element.d.ts" />
-
-class MyElement extends Polymer.Element {
-  static is = 'my-element';
+import {PolymerElement} from '@polymer/polymer';
 
+class MyElement extends PolymerElement {
   static properties = {
-    myProperty: {
-      type: String
-    }
+    myProperty: String
   };
 
   myProperty: string = 'foo';
 }
 
-customElements.define(MyElement.is, MyElement);
+customElements.define('my-element', MyElement);
 ```
 
 ### What are the performance costs?
-The additional JavaScript served for this library is aproximately 8 KB (4 KB
+The additional JavaScript served for this library is approximately 6 KB (2 KB
```

Comment:
I think it's pretty standard practice to only report the gzipped size now, but still say it's gzipped.

Is this minified too? If not, I'd run the .js file though `uglify -mc --toplevel`

Reply:
Now only reporting gzipped size. Tried to minify, but neither mishoo/UglifyJS2 nor babel/minify support ES modules (at least as far as I can tell after poking around at options for a while).

Reply:
Needed *uglify-es* for ES module support. Updated the README (0.6KB minified + gzipped).